config.toml

```diff
@@ -75,6 +75,13 @@ weight = 2
 #time_format_default = "02.01.2006"
 #time_format_blog = "02.01.2006"
 
+[languages.hi]
+title = "क्लाउड नेटिव शब्दावली"
+description = "CNCF क्लाउड नेटिव शब्दावली परियोजना का उद्देश्य क्लाउड नेटिव एप्लिकेशन के बारे में बात करते समय प्रयोग किए जाने वाले सामान्य शब्दों के संदर्भ के रूप में उपयोग किया जाना है।"
+languageName ="हिन्दी(Hindi)"
+contentDir = "content/hi"
+weight = 2
```

Comment:
@sistella 
Is this block was merged automatically by Git?
It is not problematic, but all language would be better to use different value for `weight`. (contentDir = "content/hi" , contentDir = "content/it")

Reply:
makes perfect sense! I switched it to 3, and moved it accordingly after hindi in the file

Reply:
Please note that `pt-br` team opened a PR that uses the value `3` as well.
https://github.com/cncf/glossary/pull/677/files#diff-28043ff911f28a5cb5742f7638363546311225a63eabc365af5356c70d4deb77R76-R81

IMO, it would be okay to merge this PR as-is (weight: 3 for Italiano), but it is likely that merging this PR (#685) will require the PR #677 to rebase when merging into `main`, and vice versa.

Plus, in future, we have to adjust the weights of each language if some of them are using the same weight value.